### DISCUSSION TITLE
Cannot resolve DNS

### DISCUSSION THREAD

**Discussion by ziggysdomain** [qdm12/ddns-updater#497] [Q&A]
Hi. Cannot resolve DNS - from the log. Have a NO-IP account and have entered the login credentials and the host name registered with them. Am getting:
WARN cannot DNS resolve  ******.ddns.net after 5 tries: lookup  *********.ddns.net: no such host

Any ideas where I’m going wrong? Am using the official ddns.updater app on TrueNAS-SCALE-22.12.2

**Comment by qdm12** [comment]
try changing `RESOLVER_ADDRESS`? Maybe the DNS server is not working properly. If that doesn't solve it, please share a more complete log.
